ts/components/ui/IOScrollViewWithLargeHeader.tsx

```diff
@@ -132,6 +130,8 @@ export const IOScrollViewWithLargeHeader = forwardRef<View, Props>(
           <H2
             color={theme["textHeading-default"]}
             testID={title?.testID}
+            ref={ref}
+            accessible
```

Comment:
Is this property really needed? Because in the `IOListViewWithLargeHeader` we don't set it, unlike here where we set it explicitly. By default, all `Text` components should be accessible, so maybe it's not strictly necessary.
```suggestion
```